src/main/java/sk/hfa/blog/services/BlogService.java

```diff
@@ -31,11 +34,12 @@ public BlogService(IImageService imageService, BlogArticleRepository blogArticle
     }
 
     @Override
-    public BlogArticle save(BlogArticle blogArticle) {
-        if (Objects.isNull(blogArticle))
+    public BlogArticle save(BlogArticleDto blogArticleDto) throws JsonProcessingException {
+        if (Objects.isNull(blogArticleDto))
             throw new IllegalArgumentException("Invalid blog article provided");
 
-        return blogArticleRepository.save(blogArticle);
+        Image titleImage = imageService.save(blogArticleDto.getTitleImage());
```

Comment:
Toto treba rollbacknut, anotujme to ako transactional pretoze ulozime obrazok aj v pripade ze blog sa nepodari ulozit.

Reply:
@RandomUserIK

Reply:
@Transactional